ocaml/xapi/xapi_pci.mli

```diff
@@ -26,6 +26,8 @@ val is_class_of_kind : base_class -> int -> bool
 (** Get int value for PCI {class, vendor, device}_id *)
 val int_of_id : string -> int
 
+val id_of_int : int -> string
```

Comment:
If we decide to expose this, despite being reasonably obvious, I think we should be consistent and add the docstring